elastica/contact_forces.py

```diff
@@ -0,0 +1,74 @@
+__doc__ = """ Numba implementation module containing contact between rods and rigid bodies and other rods rigid bodies or surfaces."""
+
+from elastica.typing import SystemType, AllowedContactType
+from elastica.rod import RodBase
+
+
+class NoContact:
+    """
+    This is the base class for contact applied between rod-like objects and allowed contact objects.
+
+    Notes
+    -----
+    Every new contact class must be derived
+    from NoContact class.
+
+    """
+
+    def __init__(self):
+        """
+        NoContact class does not need any input parameters.
+        """
+
+    def _order_check(
```

Comment:
Minor comments: maybe refactor `_order_check`->`_check_order` to keep consistency `<verb>_<objective>`.
Maybe `_check_system_type`?

Reply:
+1